### PR TITLE
multiple improvements to Community page

### PR DESCRIPTION
**Pull request by SethTisue**
the main motivation here was to add a link to the Scala Code of Conduct,
but then I saw a bunch of other stuff that could be improved.

review by @dickwall @heathermiller

### CODE REVIEW

**Comment by SethTisue** [comment]
I'm going to merge this since I want the CoC link up. @dickwall when you have time, a once-over would still be appreciated

File: community/index.md [hunk 1]
```diff
@@ -4,16 +4,14 @@ title: Community
...
-
-Developers in countries all over the world are using Scala for a large variety of applications across a broad range of industries. Popular ways to connect with the Scala community are via mailing lists or IRC channels, listed below.
-
-Though there are plenty of opportunities to connect face-to-face with others in the community-- for example, via local [Scala Meetups](http://scala.meetup.com/), or local [Scala user groups](http://scala.space/).
+Popular ways to connect with the Scala community include mailing lists, chat rooms, user groups, and conferences.
```
Comment (heathermiller) [inline community/index.md]: Wouldn't it be nice though to still include the link to the Scala Meetups and Scala Space website?

Comment (SethTisue) [inline community/index.md]: The Scala Space link remains below in the "User Groups" section — that seemed like enough to me, but I don't have a strong feeling about it one way or the other.

It seems I was a bit eager in removing the scala.meetup.com link — Scala Space is more curated, but now that I take a second look, I see the meetup link casts a wider net and pulls in additional relevant listings like Spark groups. So I added it back in fda1a6be8d04d3af2b98666aecca7d7741ed16b6.

Comment (heathermiller) [inline community/index.md]: Great :) Thanks a lot, Seth!

**Comment by dickwall** [comment]
All LGTM - sorry for delay, just digging through the backlog now. Thanks Seth and Heather for reviewing

**Pull request merged**